docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -129,21 +136,28 @@ researchers at Amgen, Inc, and you can read the paper in which it was published
 
 .. image:: MM_graph_time_varying_rates.jpg
 
-.. _allowing_other_complexities:
 
-Allowing other complexities
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Surfacing limitations
+~~~~~~~~~~~~~~~~~~~~~
 
 Another benefit of microsimulations in general is that they can surface complications or limitations that are obscured in non-individual-based models, such as compartmental models.
 
+We can revisit our IV iron intervention microsimulation to illustrate. The graph below depicts individual hemoglobin concentration over time, which as described earlier, is an
+important risk exposure variable in this model. When examining this curve, you may notice minute discrepancies from what you would expect to see in the real world. These are 
+limitations in our model that become more evident due to the high resolution of our microsimulation. Importantly, these same limitations could be in a population-based model, but 
+because there is no individual level detail, we wouldn't come face to face with them. 
+
+Another example that illustrates this is our microsimulation of a full-scale United States population. 350 million simulants in this population have 
+names, family members, addresses, employers, and other similar individual characteristics, and over time, they experience life events, such as migration, 
+employment change, and death. There are limitations in our model that are only visible at this level of detail, such as how we simulate migration: we assume
+that 100% of people who move change employment. Due to data availability, we have not been able to calculate a more accurate rate, but we know that this limitation
```

Comment:
I feel like there could be a better example of this, because this limitation would be pretty likely to be noticed in a less granular model. I think something about household structure would be perfect! Happy to brainstorm.

Reply:
Yeah using household structure is a good idea, could definitely use your help brainstorming the best example! Something with domestic migration maybe?